src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -285,6 +350,40 @@ public partial struct RSAParameters
         public byte[] P;
         public byte[] Q;
     }
+    [System.Runtime.InteropServices.ComVisibleAttribute(true)]
```

Comment:
I removed it. Apparently I ran the genapi tool before I had removed all of the attributes from the corresponding source file.